### ISSUE TITLE
Download Link

### ISSUE THREAD

**Issue by yanbai1993** [PKU-IMRE/VERI-Wild#16]
Hi,

Thank you for your attention to VERI-Wild dataset. 

If you download our dataset, it means you have agreed to our terms of access.

Terms of access:
1. Researcher shall use the Database only for non-commercial research and educational purposes.
2. We make no representations or warranties regarding the Database, including but not limited to warranties of non-infringement or fitness for a particular purpose.
3. Researcher accepts full responsibility for his or her use of the Database and shall defend and indemnify the VERI-Wild team, including their employees, Trustees, officers and agents, against any and all claims arising from Researcher's use of the Database, including but not limited to Researcher's use of any copies of copyrighted images that he or she may create from the Database.
4. Researcher may provide research associates and colleagues with access to the Database provided that they first agree to be bound by these terms and conditions.
5. VERI-Wild team reserves the right to terminate Researcher's access to the Database at any time.
6. If Researcher is employed by a for-profit, commercial entity, Researcher's employer shall also be bound by these terms and conditions, and Researcher hereby represents that he or she is fully authorized to enter into this agreement on behalf of such employer.

Download link for VERI-Wild:
Baidu cloud disk: Link: https://pan.baidu.com/s/1ib_gj5IKI2hSv1RbGsQmFwPassword: fbm1
Google driven: https://drive.google.com/drive/folders/1q9kClC0gy0bz2i06sHlKUh2Pqk6RxKan?usp=sharing

Download link for VERI-Wild 2.0:

VERI-Wild 2.0 has a challenging and large test set containing about 400K vehicle images that do not have any camera overlap with the training set. VERI-Wild 2.0 shares the same training set with VERI-Wild.
Baidu cloud disk: Link: https://pan.baidu.com/s/186nLRk-7X2RssK8cTDCQPw Password: qz32
Google driven: https://drive.google.com/drive/folders/1rfuIs1xSCa7CN6IKvMOzjQfbqjY-7E0h?usp=sharing

Best
Yan
